lib/src/supabase_auth.dart

```diff
@@ -88,6 +96,8 @@ class SupabaseAuth {
         }
       }
     }
+    WidgetsBinding.instance?.addObserver(_instance);
+    _instance._startDeeplinkObserver();
```

Comment:
Added `_startDeeplinkObserver` here so that the user of this package don't explicitly have to call to start deep link observer. I think it shouldn't have much of an effect for those that do not use deeplinks, but what does everyone think about this?